testdata/project-v2/PROJECT

```diff
@@ -1,4 +1,5 @@
 domain: testproject.org
+layout: go.kubebuilder.io/v2.0.0
 repo: sigs.k8s.io/kubebuilder/testdata/project-v2
```

Comment:
I think would great rebase the branch with the master. 
Note that it has a mock now for the add-on as well.